bin/node/src/aleph_cli.rs

```diff
@@ -33,6 +34,12 @@ pub struct AlephCli {
     /// with `--no-backup`, but note that that limits crash recoverability.
     #[clap(long, value_name = "PATH", group = "backup")]
     backup_path: Option<PathBuf>,
+
+    /// The maximum number of nonfinalized blocks, after which block production should be locally
+    /// stopped. DO NOT CHANGE THIS, PRODUCING MORE OR FEWER BLOCKS MIGHT BE CONSIDERED MALICIOUS
+    /// BEHAVIOUR AND PUNISHED ACCORDINGLY!
+    #[clap(long, default_value_t = 20)]
```

Comment:
This I think is still under discussion, right :) ?

Reply:
Dunno, if it's easily changeable then 20 should be alright, no?

Reply:
According to @maciejnems findings only values less than 128 are, right?

Reply:
As for me, the larger the value is the less likely we ever hit it in normal operation (lag in finalization), and thus better. On the other hand we should be below the strange threshold from the sync protocol, that Maciek discovered.

Reply:
The larger it is, the more blocks we have to roll back in an outage like the one on Wednesday. The theoretical chances of reaching a 20 finalization lag as seen through pure AlephBFT are already astronomically small (it's ~60 rounds) and the practical reasons at session startup are why this is already 20, on your suggestion. If we have any other issues with finalization we should fix them rather than masking them. We should probably also fix the session startup thing, this should be possible with the semi-recent changes to how AlephBFT acquires data, then we could drop this back to 6.

Reply:
For the record, I still think it should be larger, but let's make it 20. What I think is crucial though is to make changes to our monitoring/alerts, so that there is an alert when we start backing off (currently it would not be even visible I guess?). Could you add an appropriate task in jira?

Reply:
Very good point, I've been thinking about this but did not get to creating the task until now. Added as 1643, and this should definitely be high priority (likely relatively simple though).